pkg/util/scheme/scheme.go

```diff
@@ -0,0 +1,70 @@
+package scheme
+
```

Comment:
Please add a file-level comment here explaining what we're doing (building a singleton scheme containing all the things we're gonna care about in our project) and why (not s'posed to be using the global one, and it sucks piecing it together everywhere).